extensions/ql-vscode/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## [UNRELEASED]
```

Comment:
Can you delete this line?

Reply:
Yep — I forgot whether the automation was smart enough to add it or not.

Reply:
Deleted!